### DISCUSSION TITLE
customising data formats

### DISCUSSION THREAD

**Discussion by VilkkuKoo** [dhrubasaha08/DHT11#3] [Ideas]
so now as i got this working on my lcd display it seems to be like the reading change only like 27, 28 ... and not for example decimals 27.54, 38.9 this is no issue otherwise but the random .00 on the ends "waste" precious letters/characters. this would be a nice update to this library imo

**Comment by dhrubasaha08** [comment]
Hello @VilkkuKoo, I apologize for the late response. It was an oversight on my part as I didn't take into account that the DHT11 only provides output in whole numbers (integers), not in floating-point numbers (floats). I mistakenly used the float data type instead of the correct int data type. I will update the data type as soon as possible.

**Comment by dhrubasaha08** [comment]
Thank you for using my **DHT11** library. I really appreciate your feedback.
